x-pack/plugins/file_upload/public/components/json_index_file_picker.js

```diff
@@ -9,7 +9,6 @@ import React, { Fragment, Component } from 'react';
 import { EuiFilePicker, EuiFormRow, EuiProgress } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
-import { parseFile } from '../util/file_parser';
 
 const MAX_FILE_SIZE = 52428800;
```

Comment:
This will be by next project. I want to 
1) remove json_index_file_picker.MAX_FILE_SIZE. 
2) rename advanced setting `ml:fileDataVisualizerMaxFileSize` to `fileUpload:maxFileSize`
3) Have both ml and maps file upload use `fileUpload:maxFileSize` so they accept the same sized files. 

It has always bugged my that you could upload a 100MB CSV but are limited to only 50MB geojson file. Hopefully this will be fixed soon and all file upload sizes will be driven from a single configuration.